src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -183,19 +193,20 @@ export default function createPostGraphileHttpRequestHandler(options) {
 
     // If the user wants to see the error’s stack, let’s add it to the
     // formatted error.
+    error.stack = error.stack || '[empty stack]'
     if (options.showErrorStack)
       formattedError.stack =
-        options.showErrorStack === 'json'
+        options.showErrorStack
```

Comment:
👀 needs discussion.  `showErrorStack` is a bool everywhere else, and the lib wouldn't compile with the above conditional.  is this really a supported feature?  it looks like in both places, the stack makes it through.

im not sure of the intent.

Reply:
This was before my time, but I believe there used to be an option where you could have the stack return an array rather than a string. I don't *think* this is currently possible, but a review of the code would be needed to assert this is the case.

Reply:
ya.  it logically doesn't make sense that `showErrorStack` would affect the return type of the stack anyway.  i'm prone to change this whole logic entirely to:

`formattedError.stack = option.showErrorStack ? (error.stack || '') : ''`

Reply:
So it turns out that this is actually a documented feature: 

> `showErrorStack`: Enables adding a stack field to the error response. Can be either the boolean `true` (which results in a single stack string) or the string `'json'` (which causes the stack to become an array with elements for each line of the stack). Recommended in development, not recommended in production.
> -- https://www.graphile.org/postgraphile/usage-library/#api-postgraphilepgconfig-schemaname-options

Reply:
in the original ternary, either way, a stack was assigned.  this is a bug with the original impl.  let's file a bug and address it independently.

Reply:
i filed https://github.com/graphile/postgraphile/issues/804 for addressing this